tools/snp.sh

```diff
@@ -643,8 +662,20 @@ build_and_install_amdsev() {
   ./build.sh --package
   sudo cp kvm.conf /etc/modprobe.d/
   
-  # Install
-  cd $(ls -d snp-release-* | head -1)
+  # Get guest kernel version from the guest config file
+  local guest_kernel_version=$(get_guest_kernel_version)
+
+  # To standardize guest kernel file location across different linux distributions.
+  local bzImage_file=$(find ${SETUP_WORKING_DIR}/AMDSEV/linux/guest -name "bzImage"| head -1)
```

Comment:
Spacing. Be consistent with comment punctuation. For single statements elsewhere in comments, you don't use a period.